spec/services/csv_exporter_spec.rb

```diff
@@ -4,33 +4,40 @@
 
   # let(:responding_team)   { create :responding_team, name: 'Transport for London' }
   # let(:responder)         { create :responder, full_name: 'Joe Smith', responding_teams: [ responding_team] }
+  let(:late_team)   { create :responding_team, name: 'Transport for London' }
   let(:foi_case)          { create :closed_case, :fully_refused_exempt_s40,
                                    name: 'FOI Case name',
                                    email: 'dave@moj.com',
                                    message: 'foi message',
-                                   postal_address: nil }
+                                   postal_address: nil,
+                                   received_date: Date.new(2018, 2, 1),
+                                   date_responded: Date.new(2018, 5, 1),
+                                   external_deadline: Date.new(2018, 4, 1),
+                                   late_team_id: late_team.id
+  }
   let(:sar_case)          { create :closed_sar,
                                    name: 'SAR case name',
                                    postal_address: "2 High Street\nAnytown\nAY2 4FF",
                                    subject: 'Full details required',
                                    email: 'theresa@moj.com',
                                    message: 'my SAR message',
-                                   subject_full_name: 'Theresa Cant' }
+                                   subject_full_name: 'Theresa Cant'
+  }
 
   context 'FOI' do
     it 'returns an array of fields' do
       Timecop.freeze Time.local(2018, 10, 1, 13, 21, 33) do
         csv = CSVExporter.new(foi_case).to_csv
         expect(csv).to eq [
-                             '180830001',
+                             '180201001',
                              'FOI',
                              'closed',
                              'FOI Responding Team',
                              'foi responding user',
-                             '2018-08-30',
-                             '2018-09-13',
-                             '2018-09-27',
-                             '2018-09-25',
+                             '2018-02-01',
+                             '2018-02-15',
+                             '2018-03-01',
+                             '2018-05-01',
```

Comment:
What's the reason for changing these dates? Easier to comprehend?

Reply:
The original test asserted things from the factory. Now the test has to set them explicitly, the expectations change accordingly.